### ISSUE TITLE
Expected row count does not match observed row count

### ISSUE THREAD

**Issue by proton-code** [MIT-LCP/mimic-code#572]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.physionet.org/about/mimic/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:

* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

I've finally copied all the tables from MIMIC III according to the steps described in our tutorial website: https://mimic.physionet.org/tutorials/install-mimic-locally-ubuntu/
And when I ran the checks sql I passed almost all the tables except chartevents... and for some reason it seems that I have a duplicate chartevents table (see below)...T_T
What should I do next?  Really don't want to discard the database and start from scratch again.... any idea or suggestions?!?
Thank you so much!

psql 'dbname=mimic user=mimicuser options=--search_path=mimiciii' -f /Users/me/MIMIC3w/postgres_checks.sql
        tbl         | expected_count | observed_count | row_count_check 
--------------------+----------------+----------------+-----------------
 admissions         |          58976 |          58976 | PASSED
 callout            |          34499 |          34499 | PASSED
 caregivers         |           7567 |           7567 | PASSED
 **chartevents        |      330712483 |      661424966 | FAILED**
 cptevents          |         573146 |         573146 | PASSED
 d_cpt              |            134 |            134 | PASSED
 d_icd_diagnoses    |          14567 |          14567 | PASSED
 d_icd_procedures   |           3882 |           3882 | PASSED
 d_items            |          12487 |          12487 | PASSED
 d_labitems         |            753 |            753 | PASSED
 datetimeevents     |        4485937 |        4485937 | PASSED
 diagnoses_icd      |         651047 |         651047 | PASSED
 drgcodes           |         125557 |         125557 | PASSED
 icustays           |          61532 |          61532 | PASSED
 inputevents_cv     |       17527935 |       17527935 | PASSED
 inputevents_mv     |        3618991 |        3618991 | PASSED
 labevents          |       27854055 |       27854055 | PASSED
 microbiologyevents |         631726 |         631726 | PASSED
 noteevents         |        2083180 |        2083180 | PASSED
 outputevents       |        4349218 |        4349218 | PASSED
 patients           |          46520 |          46520 | PASSED

**Comment by tompollard** [comment]
From these numbers, it looks like you've loaded the data twice into your chartevents table. The most straightforward fix would be to drop the data in the chartevents table and then reload it, which would mean running something along the lines of:

```sql
-- drop all rows
DELETE FROM chartevents;

-- load the data
\copy CHARTEVENTS from 'CHARTEVENTS.csv' delimiter ',' csv header NULL ''
```

Is there a reason why you need to build the database on your local machine? If you'd be happy working with a cloud system, then you could use our instance on BigQuery.

There are examples for working with MIMIC in BigQuery at: https://github.com/GoogleCloudPlatform/healthcare/blob/master/datathon/mimic_eicu/tutorials/

To gain access to this database, please send an email to mimic-support@physionet.org

**Comment by proton-code** [comment]
Hi, thank you for your suggestions.... 
I did something similar (by googling how to drop a table and re-create one) and for some reason now it still did not work for me... 

Below is what I did:

mimic=# DROP TABLE mimiciii.chartevents CASCADE;

KLs-Mac:~ me$ psql -U mimicuser -d postgres
postgres-# \c mimic;

--- then I pretty much copied the step(s) related to chartevents from postgres_create_tables.sql and postgres_load_data.sql and ran them... seems to be working...

mimic=# CREATE TABLE CHARTEVENTS
mimic-# (
mimic(#   ROW_ID INT NOT NULL,
mimic(# SUBJECT_ID INT NOT NULL,
mimic(# HADM_ID INT,
mimic(# ICUSTAY_ID INT,
mimic(# ITEMID INT,
mimic(# CHARTTIME TIMESTAMP(0),
mimic(# STORETIME TIMESTAMP(0),
mimic(# CGID INT,
mimic(# VALUE VARCHAR(255),
mimic(# VALUENUM DOUBLE PRECISION,
mimic(# VALUEUOM VARCHAR(50),
mimic(# WARNING INT,
mimic(# ERROR INT,
mimic(# RESULTSTATUS VARCHAR(50),
mimic(# STOPPED VARCHAR(50),
mimic(# CONSTRAINT chartevents_rowid_pk PRIMARY KEY (ROW_ID)
mimic(# ) ;

mimic=# \copy CHARTEVENTS from '/Users/me/MIMIC3w/CHARTEVENTS.csv' delimiter ',' csv header NULL ''
COPY 330712483
---at this point it seems the table is loaded with data correctly-----

---but after I tried running the test code, it gave me the below error (in black)
KLs-Mac:~ me$ psql 'dbname=mimic user=mimicuser options=--search_path=mimiciii' -f /Users/me/MIMIC3w/postgres_checks.sql
**psql:/Users/me/MIMIC3w/postgres_checks.sql:77: ERROR:  relation "chartevents" does not exist
LINE 35: ... 'chartevents' as tbl, count(*) as row_count from chartevent...**

At this point I don't know what to say.... The data must be somewhere in my laptop... I could tell from the storage...
Maybe the best way now is to erase the whole mimic database and re-start from scratch? (I literally have done it 3 times) and still have no usable dataset at this point. 
If I want to delete everything and re-start the whole process, what should I do?

In terms of why I'm not using BigQuery, I just want to have ONE usable copy of the data somewhere at this moment so that I could start working on the project.
